rust/src/types.rs

```diff
@@ -17,7 +17,7 @@
 // TODO : Test the get_enumeration and get_structure methods
 
 use binaryninjacore_sys::*;
-use std::{fmt, mem, ptr, result, slice};
+use std::{ffi::CStr, fmt, mem, ptr, result, slice};
```

Comment:
This adds the warning

```
warning: the item `CStr` is imported redundantly
    --> src/types.rs:1606:13
     |
20   | use std::{ffi::CStr, fmt, mem, ptr, result, slice};
     |           --------- the item `CStr` is already imported here
...
1606 |         use std::ffi::CStr;
     |             ^^^^^^^^^^^^^^
     |
     = note: `#[warn(unused_imports)]` on by default
```

Could you please remove that import to resolve that warning?